hybrasyl/World.cs

```diff
@@ -1151,6 +1152,11 @@ private void ControlMessage_SpawnMonster(HybrasylControlMessage message)
             map.InsertCreature(monster);
         }
 
+        private void ControlMessage_MonolithControl(HybrasylControlMessage message)
+        {
+            
+        }
+

```

Comment:
This does nothing at all. Why is it here?

Reply:
See above comment on this.